include/verilated_timing.h

```diff
@@ -56,6 +56,36 @@
 #endif
 // clang-format on
 
+//=============================================================================
+// VlFileLine stores a SystemVerilog source code location. Used in VlCoroutineHandle for debugging
+// purposes.
+
+class VlFileLine final {
```

Comment:
```suggestion
class VlFileLineDebug final {
```

Because I suspect at some point we'll want a VlFileLine that always stores a fileline, not just when debugging.